src/torchcodec/decoders/_core/VideoDecoder.cpp

```diff
@@ -838,7 +838,7 @@ VideoDecoder::FrameBatchOutput VideoDecoder::getFramesPlayedInRange(
   return frameBatchOutput;
 }
 
-torch::Tensor VideoDecoder::getFramesPlayedInRangeAudio(
+VideoDecoder::AudioFramesOutput VideoDecoder::getFramesPlayedInRangeAudio(
```

Comment:
I decided to create a new `AudioFramesOutput` struct instead of relying on the existing `FrameOutput`, which contains unnecessary fields like `streamIndex` and `durationSeconds`. For now, those aren't needed for audio. No super strong opinion though.

Reply:
I think that's the right call, since the tensors are going to be very different.